File: src/lib/youtube.ts
/**
 * YouTube helpers — thin re-exports of `source.ts` for back-compat.
 * Prefer importing from `$lib/source` for multi-source support.
 */
export {
  isYouTubeUrl,
  normalizeYouTubeUrl,
  isSupportedUrl,
  normalizeSourceUrl,
  sourceFamily,
  UNSUPPORTED_SITE_MESSAGE,
} from "./source";
export type { SourceFamily } from "./source";

File: src-tauri/src/youtube.rs
//! YouTube helpers — re-export multi-source module for back-compat.
//! Prefer `crate::source` for new code.

pub use crate::source::{
    is_supported_url, is_youtube_url, normalize_source_url, normalize_youtube_url, source_family,
    SourceFamily, UNSUPPORTED_SITE_MESSAGE,
};
